test/test_browser.py

```diff
@@ -3025,7 +3025,7 @@ def test_sdl2_image_formats(self):
     self.btest_exit('test_sdl2_image.c', 600, args=[
       '--preload-file', 'screenshot.jpg',
       '-DSCREENSHOT_DIRNAME="/"', '-DSCREENSHOT_BASENAME="screenshot.jpg"', '-DBITSPERPIXEL=24', '-DNO_PRELOADED',
-      '-sUSE_SDL=2', '-sUSE_SDL_IMAGE=2', '-sSDL2_IMAGE_FORMATS=["jpg"]'
+      '--use-port=sdl2', '--use-port=sdl2_image?formats=jpg'
```

Comment:
I worry a little bit about use ? in command line flags since bash treats it specially.

How about `:` instead?

Reply:
Actually the `&` is even worse. :-/

Reply:
I know. That being said the syntax of some options right now (like `-sSDL2_IMAGE_FORMATS=["jpg", "png"]`) is not very shell friendly either and can require careful quoting...

Reply:
A long time ago we moved the simpler `-sFOO=bar,baz` for this reason and use and recommend that form everywhere now.

I guess this test fell through the cracks and didn't get updated..

Reply:
Updated: https://github.com/emscripten-core/emscripten/pull/21280